components/Dialogs/examples/DialogsAlertCustomizationExampleViewController.swift

```diff
@@ -40,7 +41,8 @@ class DialogsAlertCustomizationExampleViewController: MDCCollectionViewControlle
   override func viewDidLoad() {
     super.viewDidLoad()
 
-    view.backgroundColor = UIColor.white
+    view.backgroundColor =
+        containerScheme.colorScheme?.backgroundColor ?? MDCSemanticColorScheme().backgroundColor
```

Comment:
Same comment as above

Reply:
Resolving this so we can keep the conversation in one thread for all three of these.